README.md

```diff
@@ -2,6 +2,30 @@
 
 [![Build Status](https://travis-ci.org/RedHat-Israel/ROSE.svg?branch=master)](https://travis-ci.org/RedHat-Israel/ROSE)
 
+This project is a game that has been developed in order to help teaching kids python. 
+
+In this game, two race cars compete to achieve the most points.
+The race car have to recognize the race track, the obstacles, and the bonus areas;
+calculate the best path to take to avoid the pitfalls; and collect bonus points.
+The cars move autonomously on the screen within the race track game with no interference
+from the students. No joystick or mouse would be used.
+ 
+In order to control the car movements, the students needs to implement a 'driver'.
+This code is controlling the car and will decide what will be the next action of the car.
+
+For each type of obstacles there is a different action, and different points.
+
+See [examples/README](examples/README) for explanation on how to write a driver module.
+ 
+Here is a screenshot of the race UI: 
+
+![alt text](screenshot_rose.png "ROSE Race Car Game")
+
+Here is video of a race (running code from students):
+
+<a href="http://www.youtube.com/watch?feature=player_embedded&v=9rEkOMxyuhY
+" target="_blank"><img src="http://img.youtube.com/vi/9rEkOMxyuhY/0.jpg" 
+alt="ROSE Race Car Game" width="240" height="180" border="10" /></a>
```

Comment:
Can we embed the video instead of showing a link to youtube? This can also replace the screenshot.

See https://support.google.com/youtube/answer/171780?hl=en

Reply:
It seems that it is not possible:
https://stackoverflow.com/questions/11804820/embed-a-youtube-video

Reply:
So lets add an image linking to the youtube video?

A screenshot of the youtube video with the controls will be best. Use a big screenshot that fits inside the entire readme div in github.